pkg/proxy/service.go

```diff
@@ -39,18 +39,20 @@ import (
 // or can be used for constructing a more specific ServiceInfo struct
 // defined by the proxier if needed.
 type BaseServiceInfo struct {
-	clusterIP                net.IP
-	port                     int
-	protocol                 v1.Protocol
-	nodePort                 int
-	loadBalancerStatus       v1.LoadBalancerStatus
-	sessionAffinityType      v1.ServiceAffinity
-	stickyMaxAgeSeconds      int
-	externalIPs              []string
-	loadBalancerSourceRanges []string
-	healthCheckNodePort      int
-	onlyNodeLocalEndpoints   bool
-	topologyKeys             []string
+	clusterIP                         net.IP
+	port                              int
+	protocol                          v1.Protocol
+	nodePort                          int
+	loadBalancerStatus                v1.LoadBalancerStatus
+	sessionAffinityType               v1.ServiceAffinity
+	stickyMaxAgeSeconds               int
+	externalIPs                       []string
+	loadBalancerSourceRanges          []string
+	healthCheckNodePort               int
+	onlyNodeLocalEndpoints            bool
```

Comment:
as a cleanup (these names are pretty bad): how about `nodeLocalExternal` and `nodeLocalInternal` ?

Reply:
the rename could be a new commit

Reply:
done